src/graph.js

```diff
@@ -40,35 +40,79 @@ class Graph {
   // Optionally accepts an array of other GraphNodes for the new vertex to be connected to
   // Returns the newly-added vertex
   addVertex(value, edges = []) {
+    if (this.vertices.length === 1) {
+      const newNode = new GraphNode({ value, edges });
+      this.vertices.push(newNode);
 
+      // this.vertices[0]._edges = newNode;
+      this.addEdge(this.vertices[0], this.vertices[1]);
+      return newNode;
+    } 
+    const newNode = new GraphNode({ value, edges });
+    this.vertices.push(newNode);
+    return newNode;
   }
   // Checks all the vertices of the graph for the target value
   // Returns true or false
   contains(value) {
-
+    for (let i = 0; i < this.vertices.length; i++) {
+      if (this.vertices[i]._value === value) return true;
+    }
+    return false;
   }
   // Checks the graph to see if a GraphNode with the specified value exists in the graph 
   // and removes the vertex if it is found
   removeVertex(value) {
-
+    for (let i = 0; i < this.vertices.length; i++) {
+      if (this.vertices[i]._value === value) {
+        this.vertices.splice(i, 1);
+      }
+    }
   }
   // Checks the two input vertices to see if each one references the other in their respective edges array
   // Both vertices must reference each other for the edge to be considered valid
   // If only one vertex references the other but not vice versa, should not return true
   // Note: You'll need to store references to each vertex's array of edges so that you can use 
   // array methods on said arrays. There is no method to traverse the edge arrays built into the GraphNode class
   checkIfEdgeExists(fromVertex, toVertex) {
-
+    if (fromVertex._edges) {
+      for (let i = 0; i < fromVertex._edges.length; i++) {
+        if (fromVertex._edges[i] === toVertex) return true;
```

Comment:
Make sure you don't access the `_edges` and `_value` fields on the `GraphNode` class directly. Use the provided getter and setter methods for updating these fields.